compat/Makefile.am

```diff
@@ -90,11 +90,8 @@ testPreCompiler_LDFLAGS=
 #
 TESTS += testHeaders
 
-## Special Universal .h dependency test script
-## aborts if error encountered
-testHeaders: $(srcdir)/*.h $(srcdir)/os/*.h
-	$(SHELL) $(top_srcdir)/test-suite/testheaders.sh "$(CXXCOMPILE)" $^ || exit 1
+testHeaders: $(srcdir)/*.h $(srcdir)/os/*.h $(top_srcdir)/test-suite/testHeader.cc.in
+	echo testHeaders rule: PWD=$PWD
```

Comment:
AFAICT, this bare `echo` and some other new commands in other Makefile targets will produce a lot of new noise in "make check" output. Please adjust to report as much helpful information as you would like on failures and as little as possible on successes.

Reply:
removing